Userland/Libraries/LibJS/Runtime/Temporal/PlainYearMonth.cpp

```diff
@@ -30,6 +30,81 @@ void PlainYearMonth::visit_edges(Visitor& visitor)
     visitor.visit(&m_calendar);
 }
 
+// 9.5.1 ToTemporalYearMonth ( item [ , options ] ), https://tc39.es/proposal-temporal/#sec-temporal-totemporalyearmonth
+PlainYearMonth* to_temporal_year_month(GlobalObject& global_object, Value item, Object* options)
+{
+    auto& vm = global_object.vm();
+
+    // 1. If options is not present, set options to ! OrdinaryObjectCreate(null).
+    if (!options)
+        options = Object::create(global_object, nullptr);
+
+    VERIFY(options);
```

Comment:
`Object::create` can not fail (it verifies so), so there's no reason to `VERIFY` here as well if the spec doesn't ask for it.